plat/hisilicon/hikey960/hi3660_mailbox.c

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
@hzhuang1 
Are you still planning to resolve this? Would you like me to raise a tf-issues item to track?